compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4207,6 +4440,7 @@ TR::Register*
 OMR::ARM64::TreeEvaluator::vnotzEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
    return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+
```

Comment:
I don't think this empty line is needed.

Reply:
Removed the empty line.